lib/paths/path.js

```diff
@@ -220,7 +220,7 @@ class Path {
 		return {
 			statusCode: Number(statusCode),
 			headers: headers && this.generateResponseHeaders(headers),
-			body: schema ? ResponseGenerator.generate(schema) : null
+			body: schema ? ResponseGenerator.generate(schema, preferredExampleName) : null
```

Comment:
not sure the naming of schema here makes sense - it actually refers to the whole media type object?